pkgs/by-name/vm/vmctl/package.nix

```diff
@@ -0,0 +1,44 @@
+{
+  lib,
+  stdenv,
+  bash,
+  shellcheck,
+  fetchFromGitHub,
+}:
+
+stdenv.mkDerivation {
+  pname = "vmctl";
+  version = "1";
```

Comment:
Is there an actual release for this anywhere? I only see tag `v0.99`

Reply:
I think the main developer stopped doing release sometime ago. I have asked him to restart it. But for now I consider this stable, so started with version `1`.

Reply:
I see. In that case, I don't think we should mark it as version `1`. Even if the code itself happens to be stable on the main branch, being accurate in the version tagging is important for other reasons, e.g. for not notifying other distros about non-existent updates in [repology](https://repology.org/). By our [version scheme](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming), the version should be `v0.99-unstable-2024-05-14`